back/src/workout/repositories/workout.typeorm.repository.inte.ts

```diff
@@ -135,6 +138,20 @@ describe('TypeOrm Workout Repository', () => {
     expect(foundExercise).toStrictEqual(expectedWorkout)
   })
 
+  it('should find workout by program id', async () => {
+    const expectedWorkout = workoutDataBuilder({
+      id: HardCodedValuesEnum.workoutId,
+      title: 'Leg Workout',
+      programId: programFixture.id
```

Comment:
You can use HardCodedValues.programId

Reply:
You can turn them into SCREAMING_CASE to explicit the fact that it's hard coded (concention)